packages/pagination/src/Pagination.types.ts

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+export type CustomOnClick = (page: number) => void;
+
+export interface PaginationProps
+  extends Omit<React.HTMLAttributes<HTMLElement>, 'onClick'> {
+  /**
+   * `className` - additional styles, like color
+   */
+  className?: string;
+  /**
+   * `totalPages` - total number of pages
+   */
+  totalPages: number;
+  /**
+   * `currentPage` - current page
+   */
+  currentPage: number;
+  /**
+   * `renderItem` - template function to allow different components and props for PaginationItem
+   */
+  renderItem?: (item: PaginationItemProps) => JSX.Element;
+  /**
+   * `onClick` - template function for onClick event of items
+   */
+  onClick?: CustomOnClick;
+  /**
+   * `aria-label` - aria-label for root pagination component
+   */
+  'aria-label'?: string;
+  /**
+   * `getItemAriaLabel` - Function for generation of aria-label of clickable items
+   */
+  getItemAriaLabel?: getItemAriaLabelFunction;
+}
+
+export interface DefaultGetItemAriaLabelProps {
```

Comment:
Is this of line 34? I didn't quite got it, getItemAriaLabel is a function, I explain a bit more on previous comment

Reply:
as we discussed on the call, `Props` is the problem with this naming. We should use something different than `Props` because this type doesn't belong to a React component props